crates/turbo-tasks/Cargo.toml

```diff
@@ -20,6 +20,7 @@ any_key = "0.1.1"
 anyhow = "1.0.47"
 auto-hash-map = { path = "../auto-hash-map" }
 bitflags = "1.3.2"
+concurrent-queue = "1.2.2"
```

Comment:
Seems like the latest version is 2.0. Are we already using 1.2 elsewhere?

Reply:
Yep, this is used in turbo-tasks-fs and turbo-tasks-memory

I'll add an issue to upgrade the dependency